### ISSUE TITLE
 ConvertTo-PnPClientSidePage  maps wrong content

### ISSUE THREAD

**Issue by rjcg916** [SharePoint/sp-dev-modernization#8]
This commands maps Home.aspx wiki page to a Migrated_Home.aspx with wrong content.

**Comment by rjcg916** [comment]
I dug into this more and believe I found the root cause.  

On my site, I have  a SitePages library along with other Wiki libraries.  When I try to convert my SitePages/Home.aspx page, the content from "Home.aspx" on one of the Wikis appeared in the SitePages/Migrated_Home.aspx page.  I tried renaming pages across the libraries to make "Home.aspx" unique, but the command appears to always look in the first library it fines.

**Comment by jansenbe** [comment]
Good point, the tool currently defaults to the first wiki page library it finds. It essentially does not take in account that one can have multiple wiki page libraries. When you use the .Net approach you can target any library, but the PS implementation currently does not allow that. I'll create.

I've created #12 for this.

**Comment by jansenbe** [comment]
Are you OK with closing this question now that we've a bug open to fix this?
